src/APP/office/officeListItem.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './officeListItem.scss';
+
+const OfficeListItem = (props) => {

```

Comment:
OfficeListItem and OfficeGridItem are pretty much the same. It can be single component with conditional styling based on usage/view type.